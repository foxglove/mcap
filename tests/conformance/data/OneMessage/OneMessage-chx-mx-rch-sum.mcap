version https://git-lfs.github.com/spec/v1
oid sha256:efdf45b4835a26668cd4c0a020954b44039a51d7b992be65130d3ecf9b4862aa
size 263
