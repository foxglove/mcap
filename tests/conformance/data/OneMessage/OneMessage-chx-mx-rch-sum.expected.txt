Header library=s:"" profile=s:""
Schema data=b:<040506> encoding=s:c id=i:1 name=s:Example
ChannelInfo id=i:1 message_encoding=s:a metadata=m:{s:foo=s:bar} schema_id=i:1 topic=s:example
Message channel_id=i:1 data=b:<010203> log_time=n:2 publish_time=n:1 sequence=i:10
ChannelInfo id=i:1 message_encoding=s:a metadata=m:{s:foo=s:bar} schema_id=i:1 topic=s:example
SummaryOffset group_length=n:47 group_opcode=i:4 group_start=n:153
SummaryOffset group_length=n:0 group_opcode=i:13 group_start=n:200
SummaryOffset group_length=n:0 group_opcode=i:10 group_start=n:200
SummaryOffset group_length=n:0 group_opcode=i:8 group_start=n:200
Footer summary_crc=i:0 summary_offset_start=n:200 summary_start=n:153
